batching/batching.go

```diff
@@ -226,6 +232,7 @@ func (b *Batcher) handleInput(tick <-chan time.Time) (exit bool, err error) {
 			return true, nil
 		}
 		if err := b.handleData(data); err != nil {
+			// error metric? rely on logs + tracing?
```

Comment:
This would be a great place for an error metric.  Errors here should fire alerts because you have only a couple of things happening:

1.  Our code is wrong
2. APIServer is doing something unexpected